encord/configs.py

```diff
@@ -181,7 +202,21 @@ def define_headers(self, data: str) -> Dict:
         }
 
     @staticmethod
-    def from_ssh_private_key(ssh_private_key: str, password: Optional[str], **kwargs):
+    def from_ssh_private_key(ssh_private_key: str, password: Optional[str] = "", **kwargs):
+        """
+        Instantiate a UserConfig object by the content of a private ssh key.
+
+        Args:
+            ssh_private_key: The content of a private key file.
+            password: The password for the private key file.
+
+        Returns:
+            The user configuration.
```

Comment:
Not sure if this is super helpful. Maybe just say the `EncordConfig` object here. If needed, we can then add an explanation of what the `EncordConfig` is in the `__init__` method. 

What do you think?